ios/FluentUI/Command Bar/MultilineCommandBar.swift

```diff
@@ -0,0 +1,127 @@
+
+        NSLayoutConstraint.activate([
+            sheetController.view.topAnchor.constraint(equalTo: view.topAnchor),
+            sheetController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            sheetController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            sheetController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ])
+
+        bottomSheetController = sheetController
+    }
+
+    public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        if previousTraitCollection?.horizontalSizeClass != traitCollection.horizontalSizeClass || previousTraitCollection?.verticalSizeClass != traitCollection.verticalSizeClass {
+            removeRows()
+            if traitCollection.horizontalSizeClass == traitCollection.verticalSizeClass {
+                addRows(rows: &self.regularRows)
+            } else {
+                addRows(rows: &self.compactRows)
+            }
+            bottomSheetController?.preferredExpandedContentHeight = rowsStackView.systemLayoutSizeFitting(UIView.layoutFittingCompressedSize).height
+        }
+    }
+
+    @available(*, unavailable)
+    public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Private
+
+    private var rowsStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    private var compactRows: [MultilineCommandBarRow]
+
+    private var regularRows: [MultilineCommandBarRow]
+
+    private var bottomSheetController: BottomSheetController?
+
+    private func addRows(rows: inout [MultilineCommandBarRow]) {
+        for row in rows {
+            let rowView = CommandBar(itemGroups: row.itemGroups, leadingItemGroups: nil)
+            rowView.isScrollable = row.isScrollable
+            rowView.translatesAutoresizingMaskIntoConstraints = false
+
+            if row.isScrollable {
+                rowView.tokenSet[.itemBackgroundColorRest] = .uiColor {
+                    UIColor(light: GlobalTokens.neutralColor(.white),
+                            dark: GlobalTokens.neutralColor(.black))
+                }
+                rowsStackView.addArrangedSubview(rowView)
+            } else {
+                let containerView = UIView()
+                containerView.translatesAutoresizingMaskIntoConstraints = false
+                containerView.addSubview(rowView)
+
+                NSLayoutConstraint.activate([
+                    rowView.topAnchor.constraint(equalTo: containerView.topAnchor),
+                    rowView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor, constant: 16),
```

Comment:
nit: I think I might have seen a scenario like this before in our repo, but even though this is the only constant in the class would it make sense to still put it in a `Constants` struct variable? Just to remove its magic.